mapit_global/management/commands/mapit_global_import.py

```diff
@@ -203,7 +203,7 @@ def verbose(s):
                 if len(layer) != 1:
                     raise Exception("We only expect one feature in each layer")
 
-                feat = layer[1]
+                feat = next(iter(layer))
```

Comment:
Noted, sorry! I'll just use your name in future, and remove that when i next rebase.